src/components/WhyTechSVG/ValuesDriven.js

```diff
@@ -1,22 +1,23 @@
 import React from "react";
-import Grid from '@material-ui/core/Grid';
 import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme) => ({ 
     cssHeaderText: {
         fontFamily: 'Josefin Sans',
         fontStyle: 'normal',
         fontWeight: '600',
-        fontSize: '36px',
-        lineHeight: '65px',
+        fontSize: '28px',
+        lineHeight: '55px',
```

Comment:
line height gives me '44' in figma?

Reply:
@hanlinc27 Yea the line height when I set it to 44px seemed off on my end compared to figma. My goal for this pr was just to update the computer image but you're right I should just fix this issue one time.